ThunderstoreCLI/Options.cs

```diff
@@ -20,18 +50,14 @@ public abstract class PackageOptions
     [Option("package-version", SetName = "build", Required = false, HelpText = "Version number for the package")]
     public string? VersionNumber { get; set; }
 
-    [Option("output", Required = false, HelpText = "The output format for all output. Valid options are HUMAN and JSON.")]
-    public InteractionOutputType? OutputType { get; set; }
-
-    public virtual void Init()
+    public override bool Validate()
     {
-        if (OutputType.HasValue)
-            InteractionOptions.OutputType = OutputType.Value;
-    }
+        if (!base.Validate())
```

Comment:
Base returns a hardcoded `true` so this checkup is rather pointless?

Reply:
Base implementation does create the TCLI directory, that could probably be moved to Init though. I'd still leave this here in case something gets added to the base at some point.